client/jetpack-cloud/sections/pricing/header/index.tsx

```diff
@@ -44,10 +51,23 @@ const Header: React.FC = () => {
 			'Get everything your site needs, in one package — so you can focus on your business.'
 		);
 
+	// Black Friday 2020 promotion; runs Nov 20-30 automatically. Safe to remove after Dec 1
+	const promoStartDate = moment( '2020-11-20', 'YYYY-MM-DD h:mm:ss a' );
```

Comment:
@elliottprogrammer, do we know if the timeframe for the promotion depends on the user's timezone or if it's absolute?

Reply:
Hmm, that is a great question!  Reviewing the P2, it looks like the banner on jetpack.com will go live at November 20, 00:00:00 **UTC** and then go dark on November 30, 23:59:59 **UTC**.  
Hmm, now I'm trying to think about the best way to do this..  I think there is a `moment.utc()`, but that may be based on the client's current locale..  I'm going to need to look deeper into this. Please let me know if you have any ideas or suggestions.

Reply:
I think that'd work, good idea! If you manipulate all dates in UTC you shouldn't have any issue with timezones.